docs/recipient.md

```diff
@@ -0,0 +1,23 @@
+# Recipients
```

Comment:
We call this a copy recipient everywhere else so I think we should be consistent.

Reply:
@janflora done